guides/common/modules/con_limitations-of-installation-in-an-ipv6-network.adoc

```diff
@@ -3,5 +3,6 @@
 
 {Project} installation in an IPv6 network has the following limitations:
 
-* DHCP server in {RHEL} (ISC DHCP) does not provide an integration API for managing IPv6 records, therefore {SmartProxy} DHCP plug-in that provides DHCP management is limited to IPv4 subnets. In this scenario, DHCP IPv6 must be deployed as a separate unmanaged service to bootstrap client nodes into grub2, which then configures IPv6 networking either using DHCPv6 or via static IPv6 address.
+* You can install {Project} and {SmartProxies} in IPv6-only systems, dual-stack installation not supported.
```

Comment:
I'd prefer "untested, known to work" or similar. But I am fine.

Reply:
"dual-stack installation _is_ not supported."

Reply:
Thanks Suyog, fixed this, please merge